Scripts/Maps/applecave.lua

```diff
@@ -55,6 +55,9 @@ function events.LoadMap()
         elseif mon.NPC_ID  == 518 then
             mon.Hostile = false -- Buster Squeaky
             mon.Ally = 1
+            if Merc_IsAvailableForHire(526) then
```

Comment:
⚠️ Magic number detected.
Although converting everything to enums may be a pain in the ass...